cloudSupport/src/main/scala/cromwell/cloudSupport/gcp/GoogleConfiguration.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.filesystems.gcs
+package cromwell.cloudSupport.gcp
```

Comment:
Most of this gcp cloudsupport stuff was copied as is, perhaps with minimal changes. Feel free to comment though. I might feel free to fistbird.